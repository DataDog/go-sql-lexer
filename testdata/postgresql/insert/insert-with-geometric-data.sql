INSERT INTO places (name, location) VALUES ('Point Place', point '(10, 20)');
