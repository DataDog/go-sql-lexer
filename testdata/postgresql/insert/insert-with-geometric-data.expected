[
    {
        "expected": "INSERT INTO places ( name, location ) VALUES ( ?, point ? )",
        "statement_metadata": {
            "size": 12,
            "tables": [
                "places"
            ],
            "commands": [
                "INSERT"
            ],
            "comments": [],
            "procedures": []
        }
    }
]